### PR TITLE
 From philippines :)

### PR DESCRIPTION
**Pull request by jether08**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1987)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: Philippines/Thank-you.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Thank you for teaching us how to use git hub :)
```
Comment (casey-collab) [inline Philippines/Thank-you.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline Philippines/Thank-you.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
You're very welcome!

Was this meant to be a contribution to the story, or just a thank you note? If it is the second, I certainly appreciate it, but I don't want to merge it since I only want this repo to contain contributions to the story.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**